.github/workflows/cron-bookingReminder.yml

```diff
@@ -20,4 +20,4 @@ jobs:
             -X POST \
             -H 'content-type: application/json' \
             -H 'authorization: ${{ secrets.CRON_API_KEY }}' \
-            --fail
+            -sSf
```

Comment:
--silent --show-error --fail

Hide progress while showing and failing on errors